examples/acoustic_example.py

```diff
@@ -135,3 +141,6 @@ def damp_boundary(damp, h, nbpml):
 born_op = BornOperator(dm, m, src, damp, rec, t_order, spc_order)
 print("Applying")
 born_op.apply()
+
+# remove memmap file
+DenseData.remove_memmap_file()
```

Comment:
Actually I did this because I do not know if we always want to delete the file after termination, so I seperated this behaviour, I guess we can also pass this in as either a parameter or by setting a flag.